README.md

```diff
@@ -6,11 +6,12 @@ The Ubuntu Advantage client provides users with a simple mechanism to
 view, enable, and disable offerings from Canonical on their system. The
 following entitlements are supported:
 
+- [Common Criteria EAL2 certification artifacts provisioning](https://ubuntu.com/cc-eal)
```

Comment:
Can't land this branch until upstream ubuntu.com/cc-eal is made available.

Reply:
contracts.staging.canonical.com has landed the cc-eal support. We can proceed with this branch now